src/card.ts

```diff
@@ -1081,7 +1081,16 @@ export class FrigateCard extends LitElement {
             view: new View({
               view:
                 this._getConfig().view.camera_select === 'current'
-                  ? this._view.view
+                  ? // When switching to a Frigate camera,
```

Comment:
This doesn't quite match what I suggested [here](https://github.com/dermotduffy/frigate-hass-card/pull/596#issuecomment-1127763017), can't tell if that's intentional? Specifically, I was proposing that line 1094 also be included, i.e. if the user specifically asks for `clips` view in their `camera_select` parameter, then we don't let them do that _either_ (and they end up on `live`).

Something like:

```ts
const targetView = this._getConfig().view.camera_select === 'current' ? this._view.view : this._getConfig().view.camera_select;

[...]
new View({
[...]
view: View.supportsNonFrigateCameras(targetView) ? targetView : 'live'
})
```

And put a static `supportsNonFrigateCameras()` into `view.ts` as a method to determine if a view natively works with non-Frigate cameras. This would let you move your `['timeline', 'image', 'live']` check out of here and into a place I think it fits better.

Reply:
Now I get it. I wasn't sure in which situation line 1094 would be hit (as I said [here](https://github.com/dermotduffy/frigate-hass-card/pull/596#discussion_r873207524)).

Thank you, sorry for misunderstanding and I will apply the needed corrections.